kfs-purap/src/main/resources/edu/arizona/kfs/module/purap/spring-purap.xml

```diff
@@ -147,4 +147,17 @@
 
     <bean id="paymentRequestDao" parent="platformAwareDao" class="edu.arizona.kfs.module.purap.document.dataaccess.impl.PaymentRequestDaoOjb"/>
     	
+
+    <bean id="b2bPurchaseOrderService" class="edu.arizona.kfs.module.purap.document.service.impl.B2BPurchaseOrderSciquestServiceImpl">
+        <property name="b2bDao" ref="b2bDao"/>
+        <property name="requisitionService" ref="requisitionService"/>
+        <property name="parameterService" ref="parameterService"/>
```

Comment:
As far as I'm aware, you can't change the class and inherit from a parent bean at the same time.